src/nsrt_learning.py

```diff
@@ -206,8 +235,9 @@ def _update_segment_from_option_specs(segment: Segment,
     Modifies segment in place.
```

Comment:
Update this docstring, e.g. "Figure out which option was executed within the segment." is out-of-date. Make it clear that a modification is happening.

Reply:
this isn't out of date? (or it will still be true when option learning). lmk if you think it still needs changing. also it says "Modifies segment in place." in the docstring.